### PR TITLE
Make utility scripts  use Python3 and become PEP-8 compliant

### PR DESCRIPTION
**Pull request by adrianbroher**
This PR takes the various utilitiy scripts located across the source tree and makes them PEP-8 compliant according to the Flake8 configuration used in the CI linting job.

Those scripts are currently not covered by CI as the job only checks the files within the `default/python` directory.

### CODE REVIEW

File: cmake/make_versioncpp.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def execute(self, version, branch, build_no, build_sys):
         try:
             with open(self.infile) as template_file:
                 template = Template(template_file.read())
-        except:
+        except IOError:
```
Comment (Cjkjvfnby) [inline cmake/make_versioncpp.py:63]: Print error too:
```
as e: 
   print("...." % (...., e))

```

File: default/scripting/buildings/colonies/SP_ABADDONI.focs.txt [hunk 2]
```diff
@@ -64,7 +64,7 @@ BuildingType
     ]
     effectsgroups = [
         [[LIFECYCLE_MANIP_POPULATION_EFFECTS("SP_ABADDONI")]]
-
```
Comment (Cjkjvfnby) [inline default/scripting/buildings/colonies/SP_ABADDONI.focs.txt:67]: This is not python script.

**Review by Cjkjvfnby — APPROVED** [review]
Minor comments.

File: default/scripting/buildings/colonies/SP_ABADDONI.focs.txt (hunk 2 above)
Comment (adrianbroher) [reply default/scripting/buildings/colonies/SP_ABADDONI.focs.txt:67]: This was generated by a python script and the PEP-8 conformance directly impacts that generated output.

**Pull request merged**